test/e2e/support/utils.js

```diff
@@ -262,6 +262,14 @@ export async function insertBlock( searchTerm, panelName = null ) {
 	await waitForRichTextInitialization();
 }
 
+export async function convertBlock( name ) {
+	await page.mouse.move( 200, 300, { steps: 10 } );
```

Comment:
Any reason not to use move?

Reply:
If some lines of code are not needed is always best to not have them, so we have less code to maintain, and the test execute faster.
In this case I don't think this lines have any meaningful impact I'm fine with either way, I was curious if mouse.move is needed for the tests to pass, because if it is maybe we should have a comment there specifying why it is needed.

Reply:
Cool, I kept what was used before to convert a block, but I'm fine changing it later to click.